src/test/java/seedu/dengue/model/person/DateTest.java

```diff
@@ -28,14 +28,15 @@ public void isValidDate() {
         assertFalse(Date.isValidDate("")); // empty string
         assertFalse(Date.isValidDate(" ")); // spaces only
 
-        // incorrect format
-        assertFalse(Date.isValidDate("2000/01/01")); // incorrect separators
-        assertFalse(Date.isValidDate("$2000-01-01")); // special characters
-        assertFalse(Date.isValidDate("01-01-2000")); // incorrect order
+        // acceptable formats
+
```

Comment:
Would be good to include some acceptable formats, including those that you include in MESSAGE_CONSTRAINTS

Reply:
on the bottom liao